assets/js/stripe-admin.js

```diff
@@ -54,9 +54,9 @@ jQuery( function( $ ) {
 			// Toggle Payment Request buttons settings.
 			$( '#woocommerce_stripe_payment_request' ).on( 'change', function() {
 				if ( $( this ).is( ':checked' ) ) {
-					$( '#woocommerce_stripe_payment_request_button_theme, #woocommerce_stripe_payment_request_button_type, #woocommerce_stripe_payment_request_button_height' ).closest( 'tr' ).show();
+					$( '#woocommerce_stripe_payment_request_button_theme, #woocommerce_stripe_payment_request_button_type, #woocommerce_stripe_payment_request_button_size #woocommerce_stripe_payment_request_button_height' ).closest( 'tr' ).show();
```

Comment:
This also ensures that we hide it when the "Payment request" checkbox is enabled/disabled

Before:
![2021-09-22 13 47 01](https://user-images.githubusercontent.com/273592/134403582-2f0141f4-25f6-4264-9d63-0d4aa647cc49.gif)

After:
![2021-09-22 13 46 29](https://user-images.githubusercontent.com/273592/134403525-af37b446-2c42-4d23-a150-aa5efd313166.gif)